cmd/install/addon/cmd.go

```diff
@@ -242,8 +268,51 @@ func run(cmd *cobra.Command, argv []string) {
 		})
 	}
 
+	billingModel := args.billingModel
+	billingModelAccountID := args.billingModelAccountID
+
+	if !cmd.Flags().Changed(billingModelFlag) && !interactive.Enabled() {
+		interactive.Enable()
+		r.Reporter.Infof("Enabling interactive mode")
+	}
+
+	if interactive.Enabled() && !cmd.Flags().Changed(billingModelFlag) {
+		billingModel, err = interactive.GetString(interactive.Input{
+			Question: "Billing Model",
+			Help:     cmd.Flags().Lookup(billingModelFlag).Usage,
+			Default:  "standard",
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid billing model: %s", err)
+			os.Exit(1)
+		}
+	}
+
+	if !cmd.Flags().Changed(billingModelAccountIDFlag) && !interactive.Enabled() {
+		interactive.Enable()
+		r.Reporter.Infof("Enabling interactive mode")
+	}
+
+	if interactive.Enabled() && billingModel != string(v1.BillingModelStandard) &&
+		!cmd.Flags().Changed(billingModelAccountIDFlag) {
+		billingModelAccountID, err = interactive.GetString(interactive.Input{
+			Question: "Billing Account ID",
+			Help:     cmd.Flags().Lookup(billingModelAccountIDFlag).Usage,
+			Required: true,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid account id: %s", err)
```

Comment:
Do/can we validate the account id in any way?

Reply:
Not that I know of, personally, should leave this up to AMS and not do any validation here or in CS. Before the addon can be installed we will pass this information to AMS. If its not valid AMS will error when we try to consume the quota. 

Only validation i think we need here or in CS is if the billing model is not standard, then we need to pass an account id.